src/QEQ/fix_qeq.cpp

```diff
@@ -261,7 +262,10 @@ void FixQEq::allocate_matrix()
     i = ilist[ii];
     m += numneigh[i];
   }
-  m_cap = MAX((int)(m * safezone), mincap * MIN_NBRS);
+  bigint m_cap_big = MAX((int)(m * safezone), mincap * MIN_NBRS);
```

Comment:
```suggestion
  bigint m_cap_big = MAX((bigint)(m * safezone), (bigint)mincap * MIN_NBRS);
```
Shouldn't  there be casts to bigint here?